jsonschema-core.xml

```diff
@@ -3639,11 +3653,11 @@ https://example.com/schemas/common#/$defs/count/minimum
                 </t>
                 <t>
                     The main example meta-schema also restricts the usage of the Applicator
-                    vocabulary by forbidding the keywords prefixed with "unevaluated", which
+                    vocabulary by forbidding the keywords prefixed with "additional", which
```

Comment:
I'd rather not change this example.
It is still valid and is different from understanding or not understanding the unevaluated vocabulary.

Reply:
Ok. I changed it because unevaluated* is now in its own vocabulary, but it's still reasonable to use that namespace in an example. Reverting (and fixing one line because of the vocabulary separation).